### DISCUSSION TITLE
Morpheus Doca Support

### DISCUSSION THREAD

**Discussion by javiermateos** [nv-morpheus/Morpheus#1401] [Experimental]
Hello,
I'm trying to execute the [DOCA Sensitive Information Detection Example](https://github.com/nv-morpheus/Morpheus/tree/branch-23.11/examples/doca). The documentation mention that I should use the Morpheus Doca Container found in NGC, but that container does not have the environment variable `MORPHEUS_DOCA_SUPPORT` enabled, so it is not possible to run the example in that container.

I have tried to compile from source the Morpheus Container following the documentation and setting the `MORPHEUS_DOCA_SUPPORT=ON` but the environment variables [`DOCA_REPO_HOST`](https://github.com/nv-morpheus/Morpheus/blob/68c95baa9061a9b164b7e8e3a38a56305bc7a142/docker/optional_deps/doca.sh#L26) and [`DOCA_ARTIFACTS_HOST`](https://github.com/nv-morpheus/Morpheus/blob/68c95baa9061a9b164b7e8e3a38a56305bc7a142/docker/optional_deps/doca.sh#L27) must be known before to download DOCA dependencies.

I have searched the Morpheus docs and DOCA docs but I cant find the values of this environment variables. Where can I find the values they should have?

**Comment by mdemoret-nv** [comment]
@javiermateos Recently we identified a bug in our build system which prevented the `MORPHEUS_DOCA_SUPPORT` flag from propagating to the DOCA container. We have fixed that issue and pushed new images based on our `23.11.01` release today. Can you retry using the new images?

**Reply by javiermateos** [reply]
Thank you for your response @mdemoret-nv. I don't found the `23.11.01` release in Nvidia's NGC Catalog. I have checked the release on Github but I sill have the same problem.

The problem is not the compilation is about the generation of the docker container for Morpheus. Specifically, running the script [docker.sh](https://github.com/nv-morpheus/Morpheus/blob/34fa9ea5fae7158b21cebc2b17faf6a91e1402c0/docker/optional_deps/doca.sh) requires two environment variables called [`DOCA_REPO_HOST`](https://github.com/nv-morpheus/Morpheus/blob/34fa9ea5fae7158b21cebc2b17faf6a91e1402c0/docker/optional_deps/doca.sh#L26) and [`DOCA_ARTIFACTS_HOST`](https://github.com/nv-morpheus/Morpheus/blob/34fa9ea5fae7158b21cebc2b17faf6a91e1402c0/docker/optional_deps/doca.sh#L27) whose values are unknown.
